webapp/src/components/link_tooltip/tooltip.css

```diff
@@ -73,6 +73,7 @@
     line-height: 1.25;
     overflow: hidden;
     font-size: 12px;
+    word-break: break-word;
```

Comment:
Add screenshots in the PR for how the UI looked before your fix and how it looks now.